common/api-review/firestore-lite.api.md

```diff
@@ -168,8 +168,10 @@ export abstract class FieldValue {
     abstract _toFieldTransform(context: ParseContext): FieldTransform | null;
 }
 
+// Warning: (ae-forgotten-export) The symbol "FirestoreService" needs to be exported by the entry point index.d.ts

```

Comment:
I don't know how to interpret this comment. Is this a TODO? Is this suppressing a warning?

Reply:
This is an artifact from our API generation pipeline, which produces these auto-generated files. My goal is to fix this problem in the current sprint.